src/patternfly/demos/Card/templates/cardview-demo--notification-drawer.hbs

```diff
@@ -0,0 +1,51 @@
+{{#> notification-drawer}}
+  {{#> notification-drawer-body}}
+    {{#> notification-drawer-group notification-drawer-group--IsExpanded=cardview-demo--notication-drawer--IsOpen}}
+      {{#> notification-drawer-group-toggle}}
+        {{#> notification-drawer-group-toggle-title}}
+          {{#> l-flex}}
+            {{#> notification-drawer-group-toggle-title}}
```

Comment:
Could you do this in the react component? Or is this just a manipulation of the core code only?

Reply:
Yes, it's done like this

![Screen Shot 2021-03-11 at 10 20 27 AM](https://user-images.githubusercontent.com/5385435/110810113-79a9ce00-8253-11eb-86b2-94a0c8d50694.png)

Resulting in

![Screen Shot 2021-03-11 at 10 21 05 AM](https://user-images.githubusercontent.com/5385435/110810142-80384580-8253-11eb-86aa-eece18eea0c9.png)

Reply:
Nice!